[2023-11-21T00:05:08.362+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_DAG.EXTRACT_DATA_TO_S3 scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21T00:05:08.370+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_DAG.EXTRACT_DATA_TO_S3 scheduled__2023-11-20T00:00:00+00:00 [queued]>
[2023-11-21T00:05:08.370+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T00:05:08.379+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): EXTRACT_DATA_TO_S3> on 2023-11-20 00:00:00+00:00
[2023-11-21T00:05:08.384+0000] {standard_task_runner.py:57} INFO - Started process 19624 to run task
[2023-11-21T00:05:08.387+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_DAG', 'EXTRACT_DATA_TO_S3', 'scheduled__2023-11-20T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_DAG.py', '--cfg-path', '/tmp/tmph9cpi0zj']
[2023-11-21T00:05:08.390+0000] {standard_task_runner.py:85} INFO - Job 49: Subtask EXTRACT_DATA_TO_S3
[2023-11-21T00:05:08.426+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_DAG.EXTRACT_DATA_TO_S3 scheduled__2023-11-20T00:00:00+00:00 [running]> on host dfcabe2f681c
[2023-11-21T00:05:08.490+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='devyani.srivastava' AIRFLOW_CTX_DAG_ID='outcomes_DAG' AIRFLOW_CTX_TASK_ID='EXTRACT_DATA_TO_S3' AIRFLOW_CTX_EXECUTION_DATE='2023-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-20T00:00:00+00:00'
[2023-11-21T00:05:32.638+0000] {logging_mixin.py:154} INFO - formatted_object_key:  data/2023-11-20/output_2023-11-20.csv
[2023-11-21T00:05:33.211+0000] {logging_mixin.py:154} INFO - Error uploading DataFrame: An error occurred (RequestTimeTooSkewed) when calling the PutObject operation: The difference between the request time and the current time is too large.
[2023-11-21T00:05:33.222+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T00:05:33.230+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=outcomes_DAG, task_id=EXTRACT_DATA_TO_S3, execution_date=20231120T000000, start_date=20231121T000508, end_date=20231121T000533
[2023-11-21T00:05:33.295+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T00:05:33.313+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
